### PR TITLE
Small edit round after feedback from Amanda

### PR DESCRIPTION
**Pull request by janjongboom**
@AnotherButler

### CODE REVIEW

File: docs/intro-to-lora.md [hunk 1]
```diff
@@ -10,18 +10,18 @@ A typical LoRa network consists of four parts: devices, gateways, a network serv
 
 <span class="images">![Topology of a LoRa network](assets/lora1.png)<span>Topology of a LoRa network</span></span>
 
-For hardware, you need devices and gateways, similar to how you would set up a Wi-Fi network. Gateways scan the spectrum and capture LoRa packets. There is no gateway pinning here - devices are not associated with a single gateway; thus, all gateways within range of a device receive the signal. The gateways then forward their data to a network service that handles the packet.
+For hardware, you need devices and gateways, similar to how you would set up a Wi-Fi network. Gateways are simple: they just scan the spectrum and capture LoRa packets. There is no gateway pinning here - devices are not associated with a single gateway; thus, all gateways within range of a device receive the signal. The gateways then forward their data to a network service that handles the packet.
```
Comment (janjongboom) [inline docs/intro-to-lora.md:13]: Clarifying, because its fundamental to how LoRa works.

File: docs/intro-to-lora.md [hunk 2]
```diff
@@ -55,13 +55,13 @@ You also need to build devices. If you use ARM mbed (and you should), you can ei
...
 
 ### Network server
 
-For software, you need a server that understands the LoRa protocol and can interpret the data the device sends. It's possible to roll your own (Semtech can give you its reference implementation if you sign an NDA), but there are also companies building LoRa network servers as a service, handling everything on your behalf. This article discusses two such services: the Switzerland-based startup [LORIOT](https://loriot.io), and [IoT-X](http://iot-x.com) from the UK-based Stream Technologies.
+Now on to the software side. You need a server that understands the LoRa protocol and can interpret the data the device sends. It's possible to roll your own (Semtech can give you its reference implementation if you sign an NDA), but there are also companies building LoRa network servers as a service, handling everything on your behalf. This article discusses two such services: the Switzerland-based startup [LORIOT](https://loriot.io), and [IoT-X](http://iot-x.com) from the UK-based Stream Technologies.
```
Comment (janjongboom) [inline docs/intro-to-lora.md:64]: Network Server is the first part of the software requirements, so I added this first sentence.

File: docs/iotx.md [hunk 3]
```diff
@@ -48,18 +48,25 @@ Next you need a configuration file for the packet forwarder, which you can retri
...
+Now to the interesting work: building a device that can send sensor data over the LoRa network. For example, you can create a simple motion sensor using a [PIR sensor](https://www.adafruit.com/products/189) (under 10 euros at your local hardware store, and 2 euros when ordering from China). Of course, you can use any other sensor.
 
 <span class="images">![nRF51-DK, LoRa shield and a PIR sensor!](assets/lora6.jpg)<span>PIR sensor hooked up to a Nordic Semiconductor nRF51-DK with a LoRa shield</span></span>
 
 ### Some notes on writing firmware
 
-- You cannot send data constantly because of duty cycle limitations. This is a requirement of using the open spectrum. If you send too quickly, sending will fail.
-- A disadvantage of the LoRa shield is that it blocks all the pins. You can solder new headers on the back of the shield to add new peripherals or use a microcontroller such as the nRF51-DK that has the pins available twice, once through hole connectors and once through female headers.
+#### Sending data constantly
```
Comment (janjongboom) [inline docs/iotx.md:57]: Copy from loriot.md

**Comment by AnotherButler** [comment]
Looks good to me

**Pull request merged**